src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj

```diff
@@ -125,7 +125,7 @@
   <ItemGroup>
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" />
     <PackageReference Include="Microsoft.DiaSymReader" Version="$(MicrosoftDiaSymReaderVersion)" />
-    <PackageReference Include="Microsoft.CodeAnalysis.FlowAnalysis.Utilities" Version="$(MicrosoftCodeAnalysisFlowAnalysisUtilitiesVersion)" />
+    <PackageReference Include="Microsoft.CodeAnalysis.FlowAnalysis.Utilities" Version="$(MicrosoftCodeAnalysisFlowAnalysisUtilitiesVersion)" PrivateAssets="all" />
```

Comment:
I would assume this is needed because otherwise this package isn't going to work. @mavasani?

Reply:
I originally marked this as a private asset, but that led to Omnisharp's analyzer execution to fail - they add a NuGet dependency to the C# Features NuGet package but this dependency could not be found at execution time. What is our recommended approach such NuGet package references? Also tagging @rchande

Reply:
If it's needed for the code to run, it should be there. I don't know why we wouldn't include this. If you look at the dependencies for the workspaces layer, you'll see it references SQLite because it's a runtime requirement, even though it's not something any user really cares about.

Reply:
logically this is the same problem as described here https://github.com/dotnet/roslyn/pull/36237
it would be great if you could address the other one too - thanks!